spaceship/lib/spaceship/connect_api/models/app.rb

```
@@ -0,0 +1,508 @@
+        end
+      end
+
+      def get_latest_app_store_version(client: nil, platform: nil, includes: nil)
+        client ||= Spaceship::ConnectAPI
+        platform ||= Spaceship::ConnectAPI::Platform::IOS
+        filter = {
+          platform: platform
+        }
+
+        # Get the latest version
+        return get_app_store_versions(client: client, filter: filter, includes: includes)
```

RuboCop (1.89.0)

[convention] Redundant `return` detected.

(Style/RedundantReturn)